addons/core/addon/helpers/time-remaining.js

```diff
@@ -29,14 +30,13 @@ export default class extends Helper {
       difference > 0 ? Math.floor(difference / 1000) : 0;
 
     // Format the duration
-    const duration = new DurationUnitFormat(this.intl.primaryLocale, {
-      style: 'timer',
-      format: `{hours}:{minutes}:{seconds} ${this.intl.t(
```

Comment:
Hmm I forget, was there a reason we only limited it to hours before? I kind of like having more than hours

Reply:
I don't remember the reasoning but I think the new format allows for easier readability.

Reply:
I remember a discussion, I do not remember the reason for it.